Update package name, application icon for Castanets.

suyambulingamrm opened the pull request:
This change updates the package name, application name and uses
Castanets logo as application icon.

Signed-off-by: suyambu.rm <suyambu.rm@samsung.com>

sm-venugopal commented on chrome/android/BUILD.gn:
```diff
@@ -22,7 +22,7 @@ import("channel.gni")
 import("static_initializers.gni")
 
-manifest_package = "org.chromium.chrome"
+manifest_package = "com.samsung.castanets"
```
Can use "enable_castanets" and retain the original code.

Commit:
Update package name, application icon for Castanets.

This change updates the package name, application name and uses
Castanets logo as application icon.

Signed-off-by: suyambu.rm <suyambu.rm@samsung.com>

Chikdolman reviewed (approved):
LGTM

younghajung commented on chrome/android/BUILD.gn:
```diff
@@ -22,7 +22,11 @@ import("channel.gni")
 
-manifest_package = "org.chromium.chrome"
+if (enable_castanets) {
+  manifest_package = "com.samsung.castanets"
```
How about "com.samsung.android.castanets"?

suyambulingamrm replied:
Updated on latest changeset.

younghajung reviewed (approved):
LGTM
